database/migrations/2018_10_23_203754_create_campaigns_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateCampaignsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('campaigns', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('internal_title')->unique()->comment('Campaign title used internally.');
```

Comment:
I don't think we need to have a unique constraint on this field (at least, not ensured by the database).

Reply:
@katiecrane had suggested this - strong thoughts on keeping this in, @katiecrane ?

Reply:
@katiecrane thanks for the input and totally agree! I'll take it out since neither of us having strong feelings cc @DFurnes

Reply:
cool that makes sense, I'll just keep it removed then! :)